src/lib.rs

```diff
@@ -71,7 +81,15 @@ const BUFSIZE: usize = 2048;
 ///           '_   -   _'
 ///           / '-----' \
 /// ```
-pub fn say<W>(input: &str, max_width: usize, mut writer: W) -> Result<()>
+
+pub fn say<W>(input: &str, max_width: usize, writer: W) -> Result<()> 
+where
+    W: Write,
+{
+    return say_with_mood(input, max_width, writer, Moods::Natural);
```

Comment:
the last expressions of a block implicitly returns, this can be 

```suggestion
    say_with_mood(input, max_width, writer, Moods::Natural)
```